src/loki/matrix_action.cc

```diff
@@ -45,7 +45,7 @@ void check_distance(Api& request,
       if (static_cast<size_t>(path_distance) > max_timedep_distance) {
         source.set_date_time("");
         target.set_date_time("");
-        if (max_timedep_distance && !added_warning) {
```

Comment:
yes thanks! already have that change in an umerged branch somewhere, it's very annoying :D